frsca.cue

```diff
@@ -97,4 +119,59 @@ frsca: clusterPolicy?: [_]: {
             required: *true | bool
         }]
     }]
+
+// watch is used to add a tekton trigger to run a pipeline
+frsca: watch?: [Name=_]: {
```

Comment:
I like this abstraction with CUE to tie them all together! Overall looks good so far.